api/client.go

```diff
@@ -499,6 +502,15 @@ func (c *Client) SetLimiter(rateLimit float64, burst int) {
 	c.config.Limiter = rate.NewLimiter(rate.Limit(rateLimit), burst)
 }
 
+func (c *Client) Limiter() *rate.Limiter {
+	c.modifyLock.RLock()
+	c.config.modifyLock.RLock()
+	defer c.config.modifyLock.RUnlock()
+	c.modifyLock.RUnlock()
```

Comment:
Not sure it's safe to unlock here, since we're referencing c.config afterwards.  Why not just defer for both locks?

Reply:
Sure, good call. I'm embarrassed to say I just copy/pasted this implementation from one of the existing getter functions without examining it carefully. I'll go through all of the getter functions and double check this.